alacritty/src/event.rs

```diff
@@ -1344,6 +1344,16 @@ impl Processor {
                 GlutinEvent::UserEvent(Event {
                     payload: EventType::CreateWindow(options), ..
                 }) => {
+                    #[cfg(not(any(target_os = "macos", windows)))]
+                    for window_context in self.windows.values_mut() {
+                        // The window is currently current, so redraw it on the next frame as well
+                        // on Wayland, otherwise it could have the buffer of the old size, if
+                        // resize happened.
```

Comment:
```suggestion
                        // The window's context is current, so redraw it on the next frame as well
                        // on Wayland, otherwise if a resize happened it could still have the buffer of the old size.
```